### DISCUSSION TITLE
Missing IFC entries in the Blender Object properties?

### DISCUSSION THREAD

**Discussion by swiss-knight** [IfcOpenShell/IfcOpenShell#4561] [Q&A] [answered]
Hello,

in [this](https://www.youtube.com/watch?v=F7bWjwyalYs&t=20s) video, one can see a lot of different IFC related entries in the Blender Object properties area:

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/19967599/59204c80-2ed3-4c1d-9f1d-1f91e46369e2)

which are not present in Blender 4.1.1 using the BlenderBIM add-on 0.0.240402 when loading the same [file](https://github.com/myoualid/ifc-101-course/blob/main/episode-01/Resources/MAD_SCIENTIST_21.ifc):

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/19967599/97ca3895-2670-4940-9e3d-9e512991e566)

How could it be? Did I missed something? Did some stuff moved to another place since then?

Thanks! 🙏

**Comment by theoryshaw** [comment, marked as answer]
maybe...
https://youtu.be/tpQjRjB1wnU?si=FA0E9ThOFZZG-TCi

**Comment by swiss-knight** [comment]
Clever aggregation, I missed that! Thanks a lot!
It's indeed explained at [12m34](https://www.youtube.com/watch?v=tpQjRjB1wnU&t=754s) in the linked video.
